packages/@aws-cdk/aws-stepfunctions-tasks/README.md

```diff
@@ -205,6 +205,29 @@ const submitJob = new tasks.LambdaInvoke(this, 'Invoke Handler', {
 });
 ```
 
+You can also use [intrinsic functions](https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-intrinsic-functions.html) with `JsonPath.stringAt()`. 
```

Comment:
I think it's definitely a good thing that we're unblocking the use of intrinsic functions, I'm just hesitant to advertise `JsonPath.stringAt` as the entry point. However, I'm not sure if there's a better way without a full design of how to model intrinsic functions properly within the CDK. Any thoughts on a better way to present this to the user as a stopgap until that design takes place?

Reply:
@BenChaimberg right, this doesn't feel like the correct long-term solution but I think it's important to unblock this. Can we add a `todo` somewhere in the code or docs?